readthedocs/settings/base.py

```diff
@@ -343,18 +343,23 @@ def USE_PROMOS(self):  # noqa
     ES_INDEXES = {
         'project': {
             'name': 'project_index',
-            'settings': {'number_of_shards': 5,
+            'settings': {'number_of_shards': 1,
                          'number_of_replicas': 1
                          }
         },
         'page': {
             'name': 'page_index',
             'settings': {
-                'number_of_shards': 5,
-                'number_of_replicas': 1,
+                'number_of_shards': 3,
+                'number_of_replicas': 3,
+                "index": {
```

Comment:
We should put comments in the code that explain why we're doing 3 of each here.

Reply:
Added! Thanks!